utils/__init__.py

```diff
@@ -41,7 +43,37 @@
     "copasi":{},
 }
 
-def create_omex(sedml_file,sbml_file):
+def get_entry_format(file, file_type):
+    '''
+    Get the entry format for a file.
+
+    Args:
+        file_path (:obj:`str`): path to the file
+        file_type (:obj:`str`): type of the file
+
+    Returns:
+        :obj:`str`: entry format
+    '''
+    # get filepath relative to the current working directory
+    file_path = os.path.join(os.getcwd(), file)
```

Comment:
If you look at how the file paths are used in the calling create_omex functions I think they already have to be valid absolute or relative paths to the file, which should both work fine, so it shouldn't be necessary to modify the "file" argument within get_entry_format. "get filepath relative to the current working directory" implies you are trying to convert the file path into relative form but the code doesn't do that. I think we can just rename the function arg from file to file_path (as it states in the doc) and not modify it further

Reply:
resolved by https://github.com/OpenSourceBrain/SBMLShowcase/pull/18/commits/86ddb140cac7a466b7c79fafe558e828c6d84625